n3fit/src/n3fit/stopping.py

```diff
@@ -496,21 +490,22 @@ def _restore_best_weights(self):
 
     def print_current_stats(self, epoch, fitstate):
         """
-        Prints ``fitstate`` training and validation chi2s
+        Prints ``fitstate`` validation chi2 and the current total training loss
+        Note that the
         """
         epoch_index = epoch + 1
-        tr_chi2 = fitstate.total_tr_chi2()
         vl_chi2 = fitstate.total_vl_chi2()
-        total_str = f"At epoch {epoch_index}/{self.total_epochs}, total chi2: {tr_chi2}\n"
+        total_str = f"""Epoch {epoch_index}/{self.total_epochs}: loss: {fitstate.tr_loss:.7f}
+Validation loss after training step: {vl_chi2:.7f}.
+Validation chi2s: """
```

Comment:
Something went wrong here

Reply:
what do you mean? (the string is continued below)

Reply:
ah... yes... That one was continued only inside my head xD 
Thanks